strapi/strapi
# [Blocks editor] Add modifier buttons to the toolbar

### What does it do?

Adds buttons to the toolbar for bold, italic, underline, strikethrough. Clicking them should apply those styles to your current selection. You can't actually _see_ the difference yet, but it does update the state of the Slate editor.

Note that the icons do not exactly match the designs, because they're from the Remix Icons lib, which is different to what we have in `@strapi/icons`. I want to do a bit of research on the topic before addressing it with the rest of the team, but I don't think it should block this PR.

### How to test it?

Highlight part of the text. Toggle the modifier buttons, you should see their style updating.

To make sure Slate actually applies them, you can log the state by modifying the code in the BlocksEditor component:

```js
<Slate
  editor={editor}
  initialValue={initialValue}
  // add this
  onChange={() => console.log(editor.children)}
>
```
